db/fixtures/miq_policy_sets.yml

```diff
@@ -0,0 +1,8 @@
+---
+- :name: container_image_default
+  :description: Container Image default profile
+  :guid:
```

Comment:
You can just leave this out entirely, I think.  We don't want anyone to populate it by accident anyway.